CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## 2.1.0-beta.1
+ - feat: Deprecated endpoint methods are now being deprecated in generated client.
+ - feat: Adds support for WASM compiled web apps.
+ - fix: Adds validation of custom class names to look for potential collisions.
+ - fix: Only considers positional `Session` parameter when validating endpoint method.
+ - fix: Updates example documentation. 
+ - feat: Adds support for `willClose` listener on `Session`.
+ - feat: Adds stream subscriptions to message central.
+ - feat: Allows custom password hash generator in `AuthConfig`.
+ - feat: Allows rewrite rule in root path in static web directories.
+ - fix: Before a session is closed, all logging is now awaited.
+ - fix: Adds new `WebCallSession` for Relic.
+ - fix: Correctly verifies `iss` value for all possible domains in Sign in with Google.
+ - fix: Add `methodName` and `endpointName` to base session class.
+ - fix: Handles malformed web server URI parameters more gracefully.
+ - fix: Uses `text` as `KeyboardType` for validation code in `SignInWithEmailDialog`.
+ - feat: Improves error handling in `SignInWithGoogle` by rethrowing exceptions.
+ - feat: Adds support for default values in model files (types supported are `String`, `int`, `double`, `bool`, `DateTime`)
+ - feat: Adds support for nullable types in `encodeWithType` and `decodeWithType`.
+ - fix: Correctly orders logs in Insights.
+ - fix: Correctly strips data in serialization of `List` and `Map`.
+ - feat: Adds Uuid identifier to sessions.
+ - fix: Starts database pool manager on Serverpod instance creation.
+ - feat: Supports configuration through environment variables instead of yaml.
+ - fix: Adds mechanism for awaiting pending future calls on shutdown.
+ - fix: Improvements to websocket lifecycle.
+ - feat: Adds DevTools extension.
+ - chore: Updates dependencies.
+
```

Comment:
Question: I don't see streaming methods mentioned? Surely they should be part of the changelogs?